metadata/buckets.go

```diff
@@ -116,6 +116,7 @@ package metadata
 
 import (
 	digest "github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
```

Comment:
Given that `errFoundSharedLabel` is only used as a sentinel error here, I don't think we need `github.com/pkg/errors` here, and the standard `errors` package should do.

But see my other comment (wether or not we actually should be using `.Walk()` and `errFoundSharedLabel`

Reply:
:+1: Will be getting rid of errFoundSharedLabel so this import will go away.